extensions/cornerstone/src/initWADOImageLoader.js

```diff
@@ -54,20 +54,23 @@ export default function initWADOImageLoader(
       // we should set this flag to false.
       convertFloatPixelDataToInt: false,
     },
-    beforeSend: function(xhr) {
+    beforeSend: function (xhr, imagId) {
       const headers = UserAuthenticationService.getAuthorizationHeader();
 
       // Request:
       // JPEG-LS Lossless (1.2.840.10008.1.2.4.80) if available, otherwise accept
       // whatever transfer-syntax the origin server provides.
       // For now we use image/jls and image/x-jls because some servers still use the old type
       // http://dicom.nema.org/medical/dicom/current/output/html/part18.html
-      const xhrRequestHeaders = {
+      // The Accept header should only be set to a multipart/related type in
+      // wado-rs request. Other requests (e.g. wado-uri) that don't support
+      // this value in the accept header may cause the server to reject the request as unacceptable.
+      const xhrRequestHeaders = isWadoRs(imagId) ? {
```

Comment:
This is fairly ugly to have the isWadoRs know about the specific type/call.  Also, the configuration should be data source related so that one can configure it as part of hte actual configuration item, rather than globally.  Can you move this into it's own PR so that we can talk about the issue related to omitQuotationForMultipartRequest separately?

Reply:
This is fixing a bug that is caused by the code assuming the WADO Image Loader is always requesting a wado-uri. I am not sure why someone would assume this but this check ensure we only add this when wado-rs is being requested. 
Personally, I don't like this piece of code entirely. 
This code should be moved inside the loader where it belongs, having the loader need to be configured like this in every application doesn't make sense.
Any future PRs should focus on refactoring this code.